src/autocomplete.js

```diff
@@ -490,6 +487,44 @@ export default class Autocomplete extends Component {
       inputClassList.push(inputClasses)
     }
 
+    const menuClassName = `${cssNamespace}__menu`
+    const menuModifierDisplayMenu = `${menuClassName}--${displayMenu}`
+    const menuIsVisible = menuOpen || showNoOptionsFound
+    const menuModifierVisibility = `${menuClassName}--${(menuIsVisible) ? 'visible' : 'hidden'}`
+
+    const menuClassList = [
+      menuClassName,
+      menuModifierDisplayMenu,
+      menuModifierVisibility
+    ]
+
+    if (menuClasses) {
+      menuClassList.push(menuClasses)
+    }
+
+    if (menuAttributes?.className) {
+      menuClassList.push(menuAttributes.className)
+    }
+
+    if (menuAttributes?.class) {
+      menuClassList.push(menuAttributes.class)
+    }
```

Comment:
Looks like `class` takes precedence in Preact, rather than allowing both?

```suggestion
    if (menuAttributes?.class || menuAttributes?.className) {
      menuClassList.push(menuAttributes?.class || menuAttributes?.className)
    }
```

Reply:
Yeah, that's a good shout. I'll amend 😊

Reply:
Added extra test cases while I was at it.